packages/babel-preset-env/scripts/build-modules-support.js

```diff
@@ -3,23 +3,22 @@ const fs = require("fs");
 
 const moduleSupport = require("caniuse-db/features-json/es6-module.json");
 
-const acceptedWithCaveats = {
-  safari: true,
-  ios_saf: true,
-};
+const skipList = new Set(["and_chr", "android", "samsung"]);
+const acceptedWithCaveats = new Set(["safari", "ios_saf"]);
 
 const { stats } = moduleSupport;
 
 const allowedBrowsers = {};
 
 Object.keys(stats).forEach(browser => {
-  if (browser !== "and_chr") {
+  if (!skipList.has(browser)) {
     const browserVersions = stats[browser];
     const allowedVersions = Object.keys(browserVersions)
       .filter(value => {
-        return acceptedWithCaveats[browser]
+        // Edge 16/17 are marked as "y #6"
```

Comment:
isn't that usually a sign it is not fully supported? or behind a flag?

Reply:
It's a bit more nuanced.

If it's not fully supported it's marked as `a #<footnote number`.

In this case, it is fully supported, except the caveat is that it loads the file, but doesn't execute it.